desktop/app/lib/keychain/index.js

```diff
@@ -7,26 +7,17 @@ const keytar = require( 'keytar' );
  * Module constants
  */
 const keychainService = 'WordPress.com';
-const credentialsKey = 'User Credentials';
 
-function set( key, value ) {
-	return keytar.setPassword( keychainService, key, value );
+async function write( key, value ) {
+	return keytar.setPassword( keychainService, key, JSON.stringify( value ) );
 }
 
-function get( key ) {
-	return keytar.getPassword( keychainService, key );
-}
-
-async function setUserInfo( info ) {
-	return await set( credentialsKey, JSON.stringify( info ) );
-}
-
-async function getUserInfo() {
-	const info = await get( credentialsKey );
-	if ( info ) {
-		return JSON.parse( info );
+async function fetch( key ) {
```

Comment:
fetch is sorta a keyword. The way that you are using it here is fine but i'd be worried that someone might think it is a HTTP call: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

Reply:
Good call. Updated the method name to `read` instead of `fetch` in 3de8c182f7ddd880f50ccb0019fe6ef5a03b5e11. This also works better because it's more aligned with the existing `keychain.write(...)`.